src/Utilities/Compiler/Extensions/ISymbolExtensions.cs

```diff
@@ -536,6 +536,21 @@ public static bool IsImplementationOfAnyExplicitInterfaceMember(this ISymbol sym
         }
 
         public static ITypeSymbol GetMemerOrLocalOrParameterType(this ISymbol symbol)
```

Comment:
>Memer [](start = 37, length = 5)

spelling

Reply:
Thanks, I am going to fix a few more false positives in next PR, will rename the method.